salt/states/kapacitor.py

```diff
@@ -96,7 +96,7 @@ def task_present(name,
         comments.append('Task script updated')
 
     if enable:
-        if task and task['Enabled']:
+        if task and task['enabled']:
```

Comment:
It is, but if you look in the module code, I changed the return value of the module function so that it's normalized.